utils/spiel/m0_filesystem_stats

```diff
@@ -150,12 +150,25 @@ if [ $rc -ne 0 ] ; then
 fi
 }
 
+get_hostname()
+{
+        host=" "
+        if [[ -f /etc/salt/minion_id ]] ; then
+            host=$(cat /etc/salt/minion_id)
+        fi
+
+        if [ -z $host ] ; then
+           host="localhost"
```

Comment:
[Suggestion]
Keep your exiting logic as it is [i.e. host=$(hostname --fqdn) ] in absent of minion_id file. 
localhost would work for singlenode only. 
In case of multinode and deployed with only (motr + hare) we need to use actual fqdn.